### ISSUE TITLE
New license request: xlock-like [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by xsuchy** [spdx/license-list-XML#2030]
**1.** License Name: xlock-like
**2.** Short identifier: xlock-like
**3.** License Author or steward: Uknown
**4.** Comments: Found in Fedora during license review in https://gitlab.com/fedora/legal/fedora-license-data/-/issues/204
Present in CVS package.
Is this license on the SPDX License List?
The most similar license is https://spdx.org/licenses/xlock.html. The second most similar is https://spdx.org/licenses/NTP.html.
**5.** License Request Url: http://tools.spdx.org/app/license_requests/250
**6.** URL(s): http://cvs.savannah.gnu.org/viewvc/cvs/ccvs/os2/dirent.c?revision=1.3&view=markup&pathrev=cvs1-11-23#l1
**7.** OSI Status: Unknown
**8.** Example Projects: http://cvs.savannah.gnu.org/viewvc/cvs/ccvs/

**Comment by jlovejoy** [comment]
license text:

```
Permission to use, copy, modify, and distribute this software and its documentation for any purpose and without fee is hereby granted provided that the above copyright notice appears in all copies and that both the copyright notice and this permission notice appear in supporting documentation.

The author makes no representations about the suitability of this software for any purpose. This software is provided ``as is'' without express or implied warranty.
```

**Comment by jlovejoy** [comment]
This is very close to #2029  such that we might be able to have one license with markup cover both. See comparison of this license text line by line, with //text// indicating any differences with #2029 


Permission to use, copy, modify, and distribute this software 
// Permission to use, copy, modify, distribute and sell this software//

and its documentation for any purpose and without fee is hereby granted 
//and its documentation for any purpose is hereby granted without fee,//

provided that the above copyright notice appears in all copies and 
that both the copyright notice and this permission notice appear 
in supporting documentation.

The author makes no representations about the suitability of this software for any purpose. 
//Christian Michelsen Research AS makes no representations about the suitability of this software for any purpose. 

This software is provided "as is'' without express or implied warranty.
//It is provided "as is" without express or implied warranty.//

**Comment by ppisar** [comment]
Good catch. They are indeed the same sans word order.

**Comment by ppisar** [comment]
It looks so. The only difference I can see is a missing comma in "distribute and sell".

However, even if I add the comma, https://tools.spdx.org/app/check_license/ fails to recognize it. Either I missed something, or the tool cannot cope with nested "optional" elements.

**Comment by jlovejoy** [comment]
ok, for this one - the only difference I see b/w this and HPND is this one has the Oxford comma here `modify, and distribute` where HPND does not have that comma. 
and this one does NOT have a comma b/w `granted, provided` where HPND does have the comma

ugh... we do say that B.6 of the matching guidelines that punctuation matters, but I don't think these commas change the meeting

@swinslow - thoughts? I mean, not sure HPND needs more markup, but... we could make the commas optional??

**Comment by ppisar** [comment]
I spotted another difference: This one has "notice appears in" while HPND is missing the 3rd person "s" suffix in the verb.

**Comment by swinslow** [comment]
@jlovejoy For the Oxford commas, see my comments [here](https://github.com/spdx/license-list-XML/issues/2029#issuecomment-1653629649)  :)

@ppisar For the missing "s" in "appears", great catch. I'm comfortable with applying markup to treat "appear" and "appears" as the same. In fact we did that for a different HPND variant, see https://github.com/spdx/license-list-XML/blob/c2dbc5a2ada9711cc4ecbc48bc844248f711f3f2/src/HPND-sell-variant.xml#L17C94-L17C100 -- so I'm +1 to doing the same here.

**Comment by ppisar** [comment]
#2113 does not fix this.

This license has  "copyright notice appears" (the "s" letter).
This license does not have "and that the name" clause.

**Comment by ppisar** [comment]
> #2113 does not fix this.
> 
> This license has "copyright notice appears" (the "s" letter).

The "s" alterative was added to a wrong place, I will prepare a merge request.

> This license does not have "and that the name" clause.

Actually this is fixed with 424f92918dbc7b45d78e0d0598340fce23873c86 commit.

**Comment by ppisar** [comment]
> > #2113 does not fix this.
> > This license has "copyright notice appears" (the "s" letter).
> 
> The "s" alterative was added to a wrong place, I will prepare a merge request.
> 

The merge request is #2143.
